### DISCUSSION TITLE
My account is suspened

### DISCUSSION THREAD

**Discussion by qEvgeniy** [community/community#64536] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

After logging into my account i realised that it is suspened (nnow i have created a new one). I did not do anything illigal. I didn`t get any notifications or mails from github. Support is ignoring me.
I have some work and styding depending from my account and now i can't use it.
My Ticket ID: 2297856
Suspened account is pEvgeniy

**Comment by queenofcorgis** [comment, marked as answer]
Hi @qEvgeniy, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
